### PR TITLE
Update the documentation to recommend OS is installed when doing discovery

### PR DESCRIPTION
**Pull request by whowutwut**
I have been getting a lot of request and questions about the xCAT genesis kernel and why it's looping forever in the discovery process.  So to avoid this, I think the default recommendation when doing discovery and running `bmcsetup` is to go directly into provisioning the Operating system, rather than having the genesis kernel loop looks for the next destiny... until the admin does a `nodeset` to provision the OS.  

This might help alleviate some confusion..

Fix issue #1803 and #1422

### CODE REVIEW

File: perl-xCAT/xCAT/Schema.pm [hunk 1]
```diff
@@ -358,7 +358,7 @@ passed as argument rather than by table value',
...
-            chain => 'A comma-delimited chain of actions to be performed automatically when this node is discovered. ("Discovered" means a node booted, but xCAT and DHCP did not recognize the MAC of this node. In this situation, xCAT initiates the discovery process, the last step of which is to run the operations listed in this chain attribute, one by one.) Valid values:  boot, runcmd=<cmd>, runimage=<URL>, shell, standby. (Default - same as no chain - it will do only the discovery.).  Example, for BMC machines use: runcmd=bmcsetup,shell.',
+            chain => 'A comma-delimited chain of actions to be performed automatically when this node is discovered for the first time.  (xCAT and the DHCP server does not recognize the MAC address of the node and xCAT initializes the discovery process.  The last step in this process is to run the operations listed in the chain attribute, one by one.  Valid values:  boot, runcmd=<cmd>, runimage=<URL>, shell, standby. Example, to have the genesis kernel pause, chain=shell.',
```
Comment (gurevichmark) [inline perl-xCAT/xCAT/Schema.pm]: 1. In the sentence:
`(xCAT and the DHCP server does not recognize the MAC address of the node and xCAT initializes the discovery process.`
- Missing closing ")" ?
- Should "does" be "do" ?
- should "and xCAT" be "when xCAT" ?
2. In the last sentence, should "Example," be "For example" or "Example:" ?

Comment (whowutwut) [inline perl-xCAT/xCAT/Schema.pm]: Thanks, I've made the changes in another commit

**Comment by whowutwut** [comment]
@zet809  if you do not have any issues with the default documented behavior to recommend an osimage to be specified , please accept the pull request.

**Comment by zet809** [comment]
Agree to merge.

**Comment by whowutwut** [comment]
This also resolves #1788, I missed that one.

**Pull request merged**